test/smoke/features/cli/cli_install_dataindex.feature

```diff
@@ -6,12 +6,12 @@ Feature: CLI: Install Kogito Data Index
 
     When CLI install Kogito Data Index with 1 replicas
 
-    Then Kogito Data Index has 1 pods running within 5 minutes
+    Then Kogito Data Index has 1 pods running within 10 minutes
```

Comment:
why do we need 10 minutes?
data-index image startup should take less than 10s.

Reply:
we need 5 to 10 minutes because it starts also Infinispan and Kafka before starting data-index.
and on our test cluster, starting Kafka ca take some time ...

Reply:
Maybe add to the backlog a way to implement something like "and wait for kafka and data-index to be ready". Or wait for the Data Index health check to be ready, so it won't start until kafka is ready, then we could have "and wait for data-index ...."

Reply:
I don't see what it changes there. We are just waiting that data-index is in Running state.
10 minutes is just a timeout.
The operator does not spot the data-index pod before Kafka is starting ...

Reply:
before Kafka is running sorry